must-gather/collection-scripts/gather

```diff
@@ -42,6 +42,7 @@ commands+=("get pv")
 commands+=("get sc")
 commands+=("get infrastructures.config -oyaml")
 commands+=("get subscription -n openshift-storage")
+commands+=("get csv")
```

Comment:
I think that more debug info will be to get yaml output instead of only list of CSVs in text format. (Or we can do both but don't think it's necessary).
```suggestion
commands+=("get csv -o yaml")
```

Reply:
good idea - @crombus what do you think?

Reply:
the change you are asking already exists in must-gather. https://github.com/openshift/ocs-operator/blob/f6dcb364571ed3d7497a25a3698df679160e18ad/must-gather/collection-scripts/gather_ceph_resources#L115

Reply:
Cool, I didn't know it's located there. Just checked the one of the must-gather logs and it's really there: http://magna002.ceph.redhat.com/ocsci-jenkins/openshift-clusters/jnk-pr2470-b2266/jnk-pr2470-b2266_20200709T112044/logs/failed_testcase_ocs_logs_1594296567/test_write_file_to_bucket_ocs_logs/ocs_must_gather/quay-io-rhceph-dev-ocs-must-gather-sha256-a9b65f901470bcd9467474d640e230c0b1a96c9a512284be53deb6962107873e/ceph/namespaces/openshift-storage/operators.coreos.com/clusterserviceversions/ 

So then it's fine to have only `get csv`, but haven't you missed the namespace `-n openshift-storage`?

Thanks

Reply:
yes i'll add that. Thanks!